.github/workflows/bundle-size.yml

```diff
@@ -95,21 +83,9 @@ jobs:
 
             - name: Report Size
               run: |
-                  npm run report:unbundled
-                  npm run report:bundled
```

Comment:
you should still keep the `npm run report:bundled`

Reply:
We can get rid of this. It's redundant.

Reply:
oh, is that because [these lines will do the same thing](https://github.com/mParticle/mparticle-web-sdk/pull/319/commits/a07b2fb401315d2ba85182a4eb701c02539850eb#diff-749feccfd42652f4a7571f0103a5b5b516b2b6d40f64f64200ac4c64870342aaR86-R87), while also saving to the environment variable?

Reply:
yup yup yup